mifospay/src/main/java/org/mifos/mobilewallet/mifospay/password/presenter/EditPasswordPresenter.java

```diff
@@ -44,7 +44,9 @@ public void updatePassword(String currentPassword, final String newPassword,
         mEditPasswordView.startProgressBar();
         if (isNotEmpty(currentPassword) && isNotEmpty(newPassword)
                 && isNotEmpty(newPasswordRepeat)) {
-            if (isNewPasswordValid(newPassword, newPasswordRepeat)) {
+            if (currentPassword.equals(newPassword)) {
+                mEditPasswordView.showError(Constants.ERROR_PASSWORDS_CANT_BE_SAME);
```

Comment:
```suggestionm
                EditPasswordView.stopProgressBar();
                mEditPasswordView.showError(Constants.ERROR_PASSWORDS_CANT_BE_SAME);
```

Reply:
please add `EditPasswordView.stopProgressBar()` before `mEditPasswordView.showError(Constants.ERROR_PASSWORDS_CANT_BE_SAME)` so that the progress bar stops and the user is able to see the error message clearly.